template/go.mod.js

```diff
@@ -0,0 +1,36 @@
+import { File } from '@asyncapi/generator-react-sdk';
+import { GetProtocolFlags } from '../components/common';
+
+export default function({ asyncapi, params }) {
+
+  let protocolFlags = GetProtocolFlags(asyncapi)
+  let module = []
+  let dependencies = ""
+
+  if ( !protocolFlags.hasAMQP ) {
+    return
+  }
+
+  if (protocolFlags.hasAMQP) {
+    module.push("github.com/ThreeDotsLabs/watermill-amqp v1.1.2")
+  }
+
+  if (module.length > 0) {
+    dependencies = module.join("\n")
+  }
+
+  return (
+    <File name="go.mod">
+  {`
+module ${params.moduleName}
```

Comment:
What do you think about asking the user to provide a package name?

Reply:
So currently this is an option that the use can pass to the template so I am guessing you were hinting at that ?

Reply:
Yep 👍